### PR TITLE
IVS-122-Testfiles Revision 01 -- Syntax errors

### PR DESCRIPTION
**Pull request by Ghesselink**
As the name suggests, mainly syntax errors are corrected. Plus a couple of schema errors.

### CODE REVIEW

**Comment by evandroAlfieri** [comment]
ifc files look good. What are the changes to validation_handling.py and existance.py ?

File: test/files/pse001/pass-pse001-ifcpropertyset_type_check_4x3.ifc [hunk 1]
```diff
@@ -11,7 +11,7 @@ DATA;
 #4=IFCSIUNIT(*,.VOLUMEUNIT.,.MILLI.,.CUBIC_METRE.);
 #5=IFCSIUNIT(*,.TIMEUNIT.,$,.SECOND.);
 #6=IFCUNITASSIGNMENT((#5,#4,#2,#3));
-#7=IfcWallType('2nJrDaLQfJ1QPhdJR0o97J',$,$,$,$,(#8),$,$,$,$);
```
Comment (aothms) [inline test/files/pse001/pass-pse001-ifcpropertyset_type_check_4x3.ifc:14]: Do we have a check for this or did you find this by accident?

Comment (Ghesselink) [reply test/files/pse001/pass-pse001-ifcpropertyset_type_check_4x3.ifc:14]: It failed the syntax check
image

**Comment by Ghesselink** [comment]
@evandroAlfieri 

> ifc files look good. What are the changes to validation_handling.py and existance.py ?

These are changes from `development` . The PR was still targeting `main`, I've implemented that a [warning will be raised the next time](https://github.com/buildingSMART/ifc-gherkin-rules/pull/287/commits/a27466e20afcf0307d7a204fb2ed26939194b794).

File: test/files/pse001/pass-pse001-ifcpropertyset_type_check_4x3.ifc (hunk 1 above)
Comment (aothms) [reply test/files/pse001/pass-pse001-ifcpropertyset_type_check_4x3.ifc:14]: Thanks :)

**Review by civilx64 — APPROVED** [review]

**Pull request merged**